src/main/cms/cms.c

```diff
@@ -344,14 +346,20 @@ static int cmsDrawMenuItemValue(displayPort_t *pDisplay, char *buff, uint8_t row
     return cnt;
 }
 
-static int cmsDrawMenuEntry(displayPort_t *pDisplay, OSD_Entry *p, uint8_t row)
+static int cmsDrawMenuEntry(displayPort_t *pDisplay, OSD_Entry *p, uint8_t row, bool selectedRow)
 {
     #define CMS_DRAW_BUFFER_LEN 12
     #define CMS_NUM_FIELD_LEN 5
+    #define CMS_CURSOR_BLINK_DELAY_MS 500
 
     char buff[CMS_DRAW_BUFFER_LEN +1]; // Make room for null terminator.
     int cnt = 0;
 
+#ifndef USE_OSD
+    // dummy parameter access to keep the CI build from failing
+    selectedRow = false;
```

Comment:
There's `UNUSED()` for exactly that. You'd typically do this at the as an else to the conditional that uses the otherwise unused parameter.

Reply:
Interesting - thanks.